packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -219,7 +219,11 @@ async function getOutlookExePath(): Promise<string | undefined> {
   try {
     const OutlookInstallPathRegistryKey: string = `HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\OUTLOOK.EXE`;
     const key = new registry.RegistryKey(`${OutlookInstallPathRegistryKey}`);
-    const outlookExePath: string | undefined = await registry.getStringValue(key, "");
+    let outlookExePath: string | undefined = await registry.getStringValue(key, "");
+    
+    if (outlookExePath) {
+      outlookExePath = `"${outlookExePath}"`;
```

Comment:
Personally, rather than putting the string with spaces in a quote here, I would do this differently. Just return the string for the path, which can have spaces, and at the place where it is used as an argument, add the quotes around the path so it handles the command line appropriately.

Reply:
That would be in the `startDetachedProcess(sideloadFile)` on line 337. You could either always escape the argument there, or see if the implementation should have special code to check if on Windows and file has a space and then add the quotes around it. Or maybe a different command should be used to launch the process, since this doesn't need to run via cmd.exe.